tests/mir-opt/inline/inline_retag.rs

```diff
@@ -9,6 +8,8 @@ fn main() {
 
 // EMIT_MIR inline_retag.bar.Inline.after.mir
 fn bar() -> bool {
+    // CHECK-LABEL: fn bar(
+    // CHECK: (inlined foo)
```

Comment:
The point of this test is to verify that we put the `Retag` statements in the correct place. Could you add them here?